File: Back/Library-Back/Library-Back.Core/Model/Booking.cs
//Author: Martín Fernando Villa Naranjo

using System;

namespace Library_Back.Core.Model
{
    /// <summary>
    /// Booking Entity
    /// </summary>
    public class Booking
    {
        public DateTime Date_Of_Booking { get; set; }

        public DateTime Date_Of_Return { get; set; }

        public int Book_Id { get; set; }

        public string User_Document { get; set; }
    }
}

File: Front/Library-Front/src/app/models/Booking.ts
//Author: Martín Fernando Villa Naranjo

export class Booking {
    constructor(
        public Date_Of_Booking: string = "",
        public Date_Of_Return: string = "",
        public Book_Id: number = 0,
        public Booking_Status_Id: number = 0,
        public User_Document: string = ""
    ) { }
}